lib/v2/nicesss/index.js

```diff
@@ -0,0 +1,24 @@
+const utils = require('./utils');
+
+module.exports = async (ctx) => {
+    const limit = ctx.params.limit || 20;
```

Comment:
Please use the common parameter `limit` from `ctx.query.limit`.
docs: https://docs.rsshub.app/parameter#limit-entries